src/common/InfoLabel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Yeah, certainly.
Just keep in mind that we should balance between reusability and 'monster component' that tries to do too much stuff.
In that case, I think it is a good idea to have one component that does both.